phpunit/class-rest-nav-menus-controller-test.php

```diff
@@ -327,6 +329,20 @@ public function test_create_item_with_location_permission_incorrect() {
 		$this->assertErrorResponse( 'rest_cannot_assign_location', $response, rest_authorization_required_code() );
 	}
 
+	/**
+	 *
+	 */
+	public function test_create_item_with_auto_add_permission_incorrect() {
+		wp_set_current_user( self::$subscriber_id );
```

Comment:
Let's test this with a user who would be able to edit the menu, but doesn't have the `edit_theme_options` cap.